docs/pages/management/admin/daemon.mdx

```diff
@@ -143,7 +153,33 @@ To upgrade a host to a newer version of Teleport, you must:
 
 </Admonition>
 
-## Further reading
+
+## Understanding Teleport daemon command line options for unit file configuration
+
+The `teleport install systemd` command includes a number of optional flags that you can use to strictly define parameters of the generated unit file and set where that output will be written.
+The following table includes all command line options available with the `teleport install systemd` command, a brief description of what they do, and our general recommended defaults tested on **Ubuntu 20.04 LTS**:
+
+
+| Flag | Description | General Recommended Default |
+| --- | ----------- | ---- |
+| `--fd-limit` | The maximum number of open file descriptors by the service, defined by `LimitNOFILE` in the unit file. |  `--fd-limit=8192` |
+| `--env-file` | The path to the `EnvironmentFile` defined by the system unit file, containing any potential variables to be used in the configuration. | `--envfile=/etc/default/teleport` |
+| `--pid-file` | The path to the pid file containing the process identification number. | `--pid-file=/run/teleport.pid` |
+| `--teleport-path` | The full path to the `teleport` binary. | `--teleport-path=/usr/local/bin/teleport` |
+| `-o`, `--output` | The full unit file load path which will define where to create and store the system unit file itself. This flag is used to write the output to the designated file. | `--output=/etc/systemd/system/teleport.service` |
+
+If using the recommended defaults, you can enter all command line options in a single command as follows:
```

Comment:
If using the recommended defaults, then you don't need to enter all these flags manually, a simple `sudo teleport install systemd -o /etc/systemd/system/teleport.service` will do the trick.

Reply:
Mentioned this but kept the example the same, let me know if you'd like to adjust it differently